contracts/SpokePool.sol

```diff
@@ -833,6 +901,22 @@ abstract contract SpokePool is
         require(isValid, "invalid signature");
     }
 
+    function _verifySlowFill(
```

Comment:
Moving it here just for readability?

Reply:
Yep. Just to make the flow super simple and similar in each of the fill methods -- check some single-line requirements and then process the fill.